### ISSUE TITLE
The window can't be dragged if a ContentDialog is opened

### ISSUE THREAD

**Issue by MGGSK** [microsoft/WinUI-Gallery#1697]
### Which version of the app?

WinUI 3 Gallery

### Description

Steps to reproduce the bug: 
1. Open the ContentDialog sample.
2. Click the "Show dialog" button
3. Try to drag the window

### Screenshots

![Image](https://github.com/user-attachments/assets/2fc81603-bead-46a9-a546-fd5594c83ed4)

### Windows version

Windows 11 24H2 (26052)

### Additional context

This issue is probably caused by the content dialog background blocking the title bar.

**Comment by karkarl** [comment]
I wonder if this this can be repro'd with System TitleBar, since WinUI Gallery currently uses a fully custom "user control" titlebar. Hopefully this will resolve once we switch to the new TitleBar control.

**Comment by foxypiratecove37350** [comment]
Isn't that actually meant to be like this?

**Comment by MGGSK** [comment]
> Isn't that actually meant to be like this?

I think that it isn't meant to be like this because the [TitleBar](https://github.com/microsoft/microsoft-ui-xaml/blob/main/specs/TitleBar/titleBar-dev-spec.md) control in WinUI 1.7 will fix this issue.

**Comment by foxypiratecove37350** [comment]
> > Isn't that actually meant to be like this?
> 
> I think that it isn't meant to be like this because the [TitleBar](https://github.com/microsoft/microsoft-ui-xaml/blob/main/specs/TitleBar/titleBar-dev-spec.md) control in WindowsAppSDK 1.7 will fix this issue.

Well, for me it seem obvious that a modal/dialog window should block interacting with the parent window unless a specific setting/attribute/argument, as `MessageBox`es do.

I think preventing from dragging the parent window is normal, but allowing it with an attribute in the XAML could be a good thing too.

**Comment by niels9001** [comment]
Yeah, this is by design. Dialogs tend to block any other interaction of the window.
